website/cue/reference/components/sinks/splunk_hec.cue

```diff
@@ -40,11 +40,7 @@ components: sinks: splunk_hec: {
 			request: {
 				enabled:                    true
 				concurrency:                10
-				rate_limit_duration_secs:   1
 				rate_limit_num:             10
```

Comment:
This one and the above `concurrency` also seems fairly arbitrary to me. I wonder if we could just leave them at their defaults.

Reply:
Agreed. I can't find any documentation about this being necessary, only limits on the message size which is not relevant here.

Reply:
Resolved in be5ce1cae